libraries/resource_providers/api200/logical_interconnect_provider.rb

```diff
@@ -82,6 +82,33 @@ def firmware_handler(action)
         end
       end
 
+      def load_ports(item_to_update)
+        port_monitor_attributes = convert_keys(@new_resource.port_monitor, :to_s)
+        enable_port_monitor = item_to_update['portMonitor']['enablePortMonitor'] if item_to_update['portMonitor'] && item_to_update['portMonitor']['enablePortMonitor']
```

Comment:
the `&& item_to_update['portMonitor']['enablePortMonitor']` is likely not necessary and should be dropped.
Theis code:
```ruby
enable_port_monitor = item_to_update['portMonitor']['enablePortMonitor'] if item_to_update['portMonitor']
```
Should return `nil` if `'enablePortMonitor'` does not exist, and it will be overwritten bellow through the `||=`